.gitlab/build_and_test.sh

```diff
@@ -84,6 +97,12 @@ elif [ $# -eq 4 ]; then
       shift 2
     fi
   fi
+elif [ $# -eq 1 ] && [[ "$AVAILABLE_CLUSTERS" =~ "$1" ]]; then
+  VALID_CMD=true
+  SHOW_AVAILABLE=true
+elif [ $# -eq 0 ]; then
+  VALID_CMD=true
+  SHOW_AVAILABLE=true
```

Comment:
Are these blocks supposed to be identical (or am I missing the difference)? It seems like these two variables should be false in the last block.

Reply:
Note that this is the logic is the same in https://github.com/lanl/spiner/pull/110 and https://github.com/lanl/ports-of-call/pull/78

Reply:
Yes it's identical on purpose. Probably could be more compact, but maybe in the next iteration (this is a XCAP-wide change).